gen/ad_group_mu_ucn

```diff
@@ -22,6 +22,7 @@ our $A_F_BASE_DN;  *A_F_BASE_DN = \'urn:perun:facility:attribute-def:def:adBaseD
 our $A_F_GROUP_BASE_DN;  *A_F_GROUP_BASE_DN = \'urn:perun:facility:attribute-def:def:adGroupBaseDN';
 our $A_R_GROUP_NAME;  *A_R_GROUP_NAME = \'urn:perun:resource:attribute-def:def:adGroupName';
 our $A_R_GRACE_PERIOD;  *A_R_GRACE_PERIOD = \'urn:perun:resource:attribute-def:def:adGracePeriod';
+our $A_V_RM_IS_BANNED;  *A_V_RM_IS_BANNED = \'urn:perun:member_resource:attribute-def:virt:isBanned';
```

Comment:
`$A_MR_V_IS_BANNED` or `$A_MR_IS_BANNED` will suit the usual naming conventions better.

Reply:
Fixed.